src/typescript/02-ide-support/ide-support.md

```diff
@@ -24,3 +24,9 @@ Integrated Development Environments have come a long way in supporting modern we
 <a href="https://www.jetbrains.com/webstorm/download/">Webstorm</a> is a platform by JetBrains that is loved for its great code refactoring assistance and version control integration, but it does require a paid subscription.
 
 <img src="../static/img/webstorm-screenshot.png" width="100%" alt="Webstorm screenshot" />
+
+### StackBlitz
+
+<a href="https://stackblitz.com/">StackBlitz</a> is an instant fullstack web IDE for the JavaScript ecosystem. It's powered by WebContainers, the first WebAssembly-based operating system which boots the Node.js environment in milliseconds, securely within your browser tab.
```

Comment:
This feels like marketing for StackBlitz. I would remove the marketing speak and solely focus on the features (online editing, TypeScript support, etc.).